tensorflow/lite/tools/pip_package/build_pip_package_with_cmake.sh

```diff
@@ -25,6 +25,7 @@ export PACKAGE_VERSION="${TENSORFLOW_VERSION}${VERSION_SUFFIX}"
 export PROJECT_NAME=${WHEEL_PROJECT_NAME:-tflite_runtime}
 BUILD_DIR="${SCRIPT_DIR}/gen/tflite_pip/${PYTHON}"
 TENSORFLOW_TARGET=${TENSORFLOW_TARGET:-$1}
+BUILD_NUM_JOBS=4
```

Comment:
This forces it to 4, regardless if the environment sets a different value. We should only set it only if not already set (e.g., see line 19 for how `$PYTHON` is defined).

Also, for this one, can you add a comment?